cmd/machine.go

```diff
@@ -457,7 +459,17 @@ func machineCreateRequest() (*metalgo.MachineCreateRequest, error) {
 	}
 
 	if len(sshPublicKeyArgument) == 0 {
-		sshPublicKeyArgument, _ = readFromFile("~/.ssh/id_rsa.pub")
+		currentUser, err := user.Current()
+		homeDir := currentUser.HomeDir
+		defaultDir := filepath.Join(homeDir, "/.ssh/")
+		keys := []string{"id_ed25519.pub", "id_rsa.pub", "id_dsa.pub"}
```

Comment:
We should put either the allowed public key file names in a named constant, or take all files with *.pub suffix from the .ssh directory